keyboards/ferris/sweep/config.h

```diff
@@ -69,11 +56,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* serial.c configuration for split keyboard */
 #define SOFT_SERIAL_PIN D2
 
-#define EE_HANDS
-
+#define SPLIT_USB_DETECT
 /* Top left key on left half */
 #define BOOTMAGIC_LITE_ROW    0
 #define BOOTMAGIC_LITE_COLUMN 0
-/* Top right key on right half */
-#define BOOTMAGIC_LITE_ROW_RIGHT    4
-#define BOOTMAGIC_LITE_COLUMN_RIGHT 4
+
+#ifdef __AVR__
+#    define EE_HANDS
```

Comment:
What's the reason for not to using this with RP2040? Lack of toolbox support?

Reply:
I don't know how EE_HANDS can be flashed with uf2 bootloader.

Reply:
I added the option for uf2-split-* a little while ago https://github.com/qmk/qmk_firmware/pull/17257. This will also work when using transient EEPROM, it's essentially generating left/right particular firmwares.

Reply:
I'm afraid that having to build distinct left and right firmware for RP2040 may generate even more support questions in the Sweep channel.